.gitignore

```diff
@@ -10,3 +10,8 @@ __pycache__/
 **/demo_extract_features.png
 **/demo_clip_features*.png
 
+# Datasets
+datasets/
```

Comment:
I would avoid saving data inside code bases even with gitignore. Better to save in a centralized location.

Reply:
Discussed on Slack, user can select where to save themselves